Test/ServerTelemetryChannel.Test/Shared.Tests/SamplingTelemetryProcessorTest.cs

```diff
@@ -15,6 +15,8 @@
     [TestClass]
     public class SamplingTelemetryProcessorTest
     {
+        Random random = new Random();
+
```

Comment:
Given that SamplingProcessor is modified to parse included/excluded types flags from user input- please ensure unit tests already cover this, or add new tests.